geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -378,6 +379,16 @@ public StoppableCountDownLatch getInitializationLatchAfterGetInitialImage() {
 
   private final EntryEventFactory entryEventFactory;
   private final RegionMapConstructor regionMapConstructor;
+  private final Timer createTimer;
```

Comment:
Thank you for picking better names than the original `putCreateTimer` and `putPutTimer`, etc